spyql/parser.py

```diff
@@ -20,11 +22,8 @@
 
 
 def get_agg_funcs():
-    import spyql.agg
-    import inspect
```

Comment:
PEP8 says "Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants." cf. https://peps.python.org/pep-0008/#imports